dist/scss/wds-typography/placeholders/_text-style.scss

```diff
@@ -0,0 +1,26 @@
+%wds-font-style-italic {
+	font-style: italic;
+}
+
+%wds-font-style-normal {
+	font-style: normal;
+}
+
+%wds-text-transform-uppercase {
+	text-transform: uppercase;
+}
+
+%wds-text-transform-lowercase {
+	text-transform: lowercase;
+}
+
+%wds-text-transform-none {
+	text-transform: none;
+}
+
+%wds-midlight {
```

Comment:
It won't work as expected for multi-line, you need to add `box-decoration-break: clone;`
see https://github.com/Wikia/f2/search?q=box-decoration-break&unscoped_q=box-decoration-break
see https://developer.mozilla.org/en-US/docs/Web/CSS/box-decoration-break

Reply:
@vforge we tested it and it works: 
![screen shot 2018-07-25 at 15 47 23](https://user-images.githubusercontent.com/1534495/43204803-23559280-9022-11e8-844d-2653ff54d9e3.png)
the only case we are aware it does not work and `box-decoration-break: clone;` doesn't help either is when parent element uses `display: flex`

Reply:
btw, we should use % in the gradient instead of px to support all font sizes

Reply:
@rybmat There's 3px padding on the left on the 1st line and not on the right side of the 1st line. There's no padding on the 2nd line and on the left side on the 3rd line. The right side on the 3rd line has a padding.

Reply:
obviously I'm blind ;)

Reply:
I added the rule, thanks @vforge